docs/README.md

```diff
@@ -50,7 +50,6 @@ Licensed under the [Apache License, Version 2.0].
 
 [gitbook]: https://www.gitbook.com/
 
-[what is electrode native]: ./overview/what-is-ern.md
+[What is electrode native]: ./overview/what-is-ern.md
```

Comment:
Also capitalize "Electrode Native"?

Reply:
Oops wait, I just realized it refers to a markdown link. Ignore my comment

Reply:
Actually, it still applies. The link target also has "Electrode Native" capitalized. Would be good to keep it the same.

Reply:
👍